src/OpenTelemetry.Api/Trace/ActivityExtensions.cs

```diff
@@ -0,0 +1,57 @@
+﻿// <copyright file="ActivityExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics;
+using System.Linq;
+
+namespace OpenTelemetry.Trace
+{
+    /// <summary>
+    /// Extension methods on Activity.
+    /// </summary>
+    public static class ActivityExtensions
+    {
+        /// <summary>
+        /// Sets the status of activity execution.
+        /// Activity class in .NET does support 'Status'.
+        /// This extension provides a workaround to store Status as special tags with key name of ot.status_code and ot.status_descriptionkeys.
```

Comment:
the keynames in comment are not matching actual code which use ot.status_code and ot.status_description